src/components/Swap/index.tsx

```diff
@@ -73,10 +74,9 @@ export default function Swap(props: SwapProps) {
           <SwapInfoProvider routerUrl={props.routerUrl}>
             <Input disabled={isDisabled} focused={focused} />
             <ReverseButton disabled={isDisabled} />
-            <Output disabled={isDisabled} focused={focused}>
-              <Toolbar />
-              <SwapActionButton disabled={isDisabled} />
-            </Output>
+            <Output disabled={isDisabled} focused={focused} />
```

Comment:
What happened to the Toolbar?

Reply:
![image](https://user-images.githubusercontent.com/5773490/194935047-fbea62f1-1265-4b6e-830c-790aeea5c142.png)